### PR TITLE
feat: hold traces tooltip on hover

### PR DESCRIPTION
**Pull request by palbizu**
## Description
Hold tooltip on hover

### Testing
Locally tested

### Checklist:
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Any dependent changes have been merged and published in downstream modules

### Documentation
Make sure that you have documented corresponding changes in this repository or [hypertrace docs repo](https://github.com/hypertrace/hypertrace-docs-website) if required.

### CODE REVIEW

File: projects/observability/src/shared/components/cartesian/d3/chart/cartesian-chart.ts [hunk 1]
```diff
@@ -608,16 +609,23 @@ export class DefaultCartesianChart<TData> implements CartesianChart<TData> {
...
-    this.eventListeners.forEach(listener => {
-      if (listener.event === ChartEvent.MouseLeave) {
-        listener.onEvent();
-      }
-    });
-
-    this.hideCrosshair();
+    this.tooltip?.hovered$
+      .pipe(
+        debounceTime(300),
```
Comment (anandtiwary) [inline projects/observability/src/shared/components/cartesian/d3/chart/cartesian-chart.ts:614]: Why do we need the debounceTime?

Comment (palbizu) [reply projects/observability/src/shared/components/cartesian/d3/chart/cartesian-chart.ts:614]: Added to provide a better user experience when hover over the graph with the cursor

File: projects/observability/src/shared/components/utils/chart-tooltip/chart-tooltip-popover.ts [hunk 2]
```diff
@@ -30,8 +33,8 @@ export class ChartTooltipPopover<TData, TContext> implements ChartTooltipRef<TDa
       this.popoverRef.updatePositionStrategy({
         type: PopoverPositionType.FollowMouse,
         boundingElement: element,
-        offsetX: 20,
-        offsetY: 30
+        offsetX: 0,
```
Comment (anandtiwary) [inline projects/observability/src/shared/components/utils/chart-tooltip/chart-tooltip-popover.ts:36]: What would be the impact of this change at other places?

Comment (palbizu) [reply projects/observability/src/shared/components/utils/chart-tooltip/chart-tooltip-popover.ts:36]: will be changing the way that we display the tooltip keeping it visible when the user has the cursor on the tooltip but will not break the functionality.

**Pull request closed without merging**